mycli/packages/filepaths.py

```diff
@@ -3,11 +3,13 @@
 from mycli.encodingutils import text_type
 import os
 
+DEFAULT_SOCKET_DIRS = ('/var/run/', '/var/lib/', '/tmp')
```

Comment:
`/tmp` might not be such a good idea; on a multi-user system anyone might create a socket in this directory to intercept passwords...